common/thorhelper/thorcommon.cpp

```diff
@@ -1823,14 +1823,26 @@ void setAutoAffinity(unsigned curProcess, unsigned processPerMachine, const char
     unsigned numaMap[NUMA_NUM_NODES];
     unsigned numNumaNodes = 0;
 #if defined(LIBNUMA_API_VERSION) && (LIBNUMA_API_VERSION>=2)
-    for (unsigned i=0; i<=numa_max_node(); i++)
+    unsigned maxcpus = numa_num_configured_cpus();
```

Comment:
numa_num_configured_cpus() could return with an incorrect value if cpus are disabled.  
Why not numa_num_task_cpus() here ?